Userland/Applications/Piano/WaveWidget.cpp

```diff
@@ -14,6 +14,7 @@
 
 WaveWidget::WaveWidget(TrackManager& track_manager)
     : m_track_manager(track_manager)
+    , m_samples(FixedArray<Audio::Sample>::must_create_but_fixme_should_propagate_errors(sample_count))
```

Comment:
I've changed it to a `Vector`. For now it is in a separate commit in case this is not what you meant/intended. If you want me to rebase it into the original commit just let me know.